### ISSUE TITLE
Header collapses

### ISSUE THREAD

**Issue by tvdijen** [OpenConext/OpenConext-profile#112]
Before:

![afbeelding](https://user-images.githubusercontent.com/841045/50234082-2e2a5180-03b5-11e9-89d3-8df9643ab64c.png)

After:

![afbeelding](https://user-images.githubusercontent.com/841045/50234099-3a161380-03b5-11e9-93a7-6efaea260187.png)

**Comment by thijskh** [comment]
Thanks. Is this only in IE? Which version(s)?

**Comment by tvdijen** [comment]
Yeah, I can't reproduce it on FF, Chrome or Edge, so it appears to be an IE-only issue.
In this case, I was using IE11
